### PR TITLE
Adds green color to new note button

### PR DESCRIPTION
**Pull request by geekyd**
Changes: 
- Adds green color to new note button for better visibility 

Screenshot:
screenshot from 2017-10-07 19-25-42

Fixes: #493

### CODE REVIEW

File: public/views/index/body.ejs [hunk 1]
```diff
@@ -13,14 +13,14 @@
                             </li>
                             <div class="ui-signin" style="float: right; margin-top: 8px;<% if(signin) { %> display: none;<% } %>">
                                 <% if(allowAnonymous) { %>
-                                <a type="button" href="<%- url %>/new" class="btn btn-sm btn-link"><i class="fa fa-plus"></i> <%= __('New guest note') %></a>
+                                <a type="button" href="<%- url %>/new" class="btn btn-sm btn-success"><i class="fa fa-plus"></i> <%= __('New guest note') %></a>
```
Comment (SISheogorath) [inline public/views/index/body.ejs]: I'm not sure if we should use this for not-singed in people, too. As you can see below the sign-in button already uses `btn-success` and from my perspective, there shouldn't be two buttons with this color

**Comment by SISheogorath** [comment]
In generell how about using `btn-primary` instead of `btn-success` for creating new notes? I'm not sure about the UX for the front page when you are a not signed in person. Because the "all features" button uses this color, too. But in general, it's maybe useful because it bites with the sign-in button.

File: public/views/index/body.ejs (hunk 1 above)
Comment (geekyd) [reply public/views/index/body.ejs]: @SISheogorath The purpose of making these changes is to highlight the `new note` button. Now the button is green, which makes it easier for the user to spot it.

**Comment by geekyd** [comment]
@SISheogorath Using `btn-primary` seems a better choice.

File: public/views/index/body.ejs (hunk 1 above)
Comment (geekyd) [reply public/views/index/body.ejs]: @SISheogorath I have changed the color please review.

screenshot from 2017-10-07 23-21-34

**Comment by ccoenen** [comment]
Thanks for taking the time!

**Comment by SISheogorath** [comment]
Updated Screenshot:

image


I like it! 👍

**Pull request merged**